File: Back-end/RoomSocialBE/DTOs/SearchDTO.cs
namespace RoomSocialBE.DTOs
{
    public class SearchDTO
    {
        public string? SearchName { get; set; }
        public string? CategoryName { get; set; }
        public string SortBy { get; set; } = "create_date";
        public bool SortDescending { get; set; } = true;
        public int? From { get; set; }
        public int? To { get; set; }
        public double? ArceFrom { get; set; }
        public double? ArceTo { get; set; }
        public string? Address { get; set; }
    }
}

File: RoomSocialFE/Models/search.dto.ts
export interface SearchDTO {
  SearchName?: string;
  CategoryName?: string;
  SortBy?: string;
  SortDescending?: boolean;
  From?: number | null;
  To?: number | null;
  ArceFrom?: number;
  ArceTo?: number;
  Province?: string;
  District?: string;
  Ward?: string;
}
